src/extension/importer/core/src/main/java/org/geoserver/importer/Importer.java

```diff
@@ -987,6 +1010,11 @@ void doIndirectImport(ImportTask task) throws IOException {
                         resource.setLatLonBoundingBox(cb.getLatLonBounds(nativeBounds,
                                 resource.getCRS()));
                         getCatalog().save(resource);
+                        
+                        //Do not re-calculate on subsequent imports
+                        if (resource.getMetadata().get("recalculate-bounds").equals(Boolean.TRUE)) {
```

Comment:
Hum... wondering, shall we really keep this one around "forever"? Can't we remove it instead?